lms/djangoapps/teams/static/teams/js/teams_features.js

```diff
@@ -4,38 +4,42 @@
 
     var xblock;
 
-    function createRocketChatDiscussion(urlRocketChat){
-
+    function createReplacementComponent(replacementComponentUrl){
```

Comment:
Please add docstrings for the javascript methods as well, is there any good example of docstring on the existing javascript edx-platform code?